frontend/src/toolbar/flags/featureFlagsLogic.ts

```diff
@@ -102,14 +102,18 @@ export const featureFlagsLogic = kea<featureFlagsLogicType>({
     events: ({ actions }) => ({
         afterMount: () => {
             actions.getUserFlags()
-            ;(window['posthog'] as PostHog).onFeatureFlags((_, variants) => {
-                if (variants) {
-                    toolbarLogic.actions.updateFeatureFlags(variants)
+            if (window) {
+                ;(window['posthog'] as PostHog).onFeatureFlags((_, variants) => {
```

Comment:
This will result in the same error. We need to test `window['posthog']`

Reply:
Yep good catch, thank you!